src/assets/docs/azure-batch-tutorial.md

```diff
@@ -0,0 +1,44 @@
+1. Create a compute pool. Note that this pool is not setup to automatically resize. You may also need to pick a larger VM size with a larger dataset. 
+    ```
+    $ ./shipyard pool add --configdir config.dockstore.md5sum
+    ```
+1. Submit the job and watch the output (this should take roughly a minute if the pool already exists)
+    ```
+    $ ./shipyard jobs add --configdir config.dockstore.md5sum --tail stdout.txt
+    2017-05-24 14:19:21.543 INFO - Adding job dockstorejob to pool dockstore
+    2017-05-24 14:19:21.989 INFO - uploading file /tmp/tmp7lgz7_j7 as 'shipyardtaskrf-dockstorejob/dockertask-00012.shipyard.envlist'
+    2017-05-24 14:19:22.027 DEBUG - submitting 1 tasks (0 -> 0) to job dockstorejob
+    2017-05-24 14:19:22.090 INFO - submitted all 1 tasks to job dockstorejob
+    2017-05-24 14:19:22.090 DEBUG - attempting to stream file stdout.txt from job=dockstorejob task=dockertask-00012
+    Creating directories for run of Dockstore launcher at: ./datastore//launcher-e849c691-cc47-4bfa-a443-b8830794ae0a
+    Provisioning your input files to your local machine
+    Downloading: #input_file from https://raw.githubusercontent.com/briandoconnor/dockstore-tool-md5sum/master/md5sum.input into directory: /mnt/batch/tasks/workitems/dockstorejob/job-1/dockertask-00012/wd/./datastore/launcher-e849c691-cc47-4bfa-a443-b8830794ae0a/inputs/ce735ade-8c46-4736-a7d8-2fc0cb7d2e87
+    [##################################################] 100%
+    Calling out to cwltool to run your tool
+    ...
+    Final process status is success
+    
+    Saving copy of cwltool stdout to: /mnt/batch/tasks/workitems/dockstorejob/job-1/dockertask-00012/wd/./datastore/launcher-e849c691-cc47-4bfa-a443-b8830794ae0a/outputs/cwltool.stdout.txt
+    Saving copy of cwltool stderr to: /mnt/batch/tasks/workitems/dockstorejob/job-1/dockertask-00012/wd/./datastore/launcher-e849c691-cc47-4bfa-a443-b8830794ae0a/outputs/cwltool.stderr.txt
+    
+    Provisioning your output files to their final destinations
+    Uploading: #output_file from /mnt/batch/tasks/workitems/dockstorejob/job-1/dockertask-00012/wd/./datastore/launcher-e849c691-cc47-4bfa-a443-b8830794ae0a/outputs/md5sum.txt to : s3://dockstore.temp/md5sum.txt
+    Calling on plugin io.dockstore.provision.S3Plugin$S3Provision to provision to s3://dockstore.temp/md5sum.txt
+    [##################################################] 100%
+    ```
+1. You can repeat the process with `config.dockstore.bwa` which is a more realistic bioinformatics workflow from the [PCAWG project](http://icgc.org/working-pancancer-data-aws) and takes roughly seven hours. 
```

Comment:
All of the ordered bullet points  in this doc seem to be number 1.

Reply:
It's actually a bit of a amusing trick. Markdown actually doesn't care about the numbers, so you can use 1. or anything really to save time. 
https://daringfireball.net/projects/markdown/syntax#list